### DISCUSSION TITLE
Is there a way to directly upscale the generated image?

### DISCUSSION THREAD

**Discussion by Eleksar387** [runew0lf/RuinedFooocus#63] [Q&A] [answered]
It seems that if I want to use upscale, I first have to create the image, go to the output folder and then drag the image in the upscale box. Isn't there an option to generate and then immediately upscale that image without the need for the manual steps?

**Comment by yownas** [comment, marked as answer]
At the moment, no. You should be able to drag-n-drop it from the gallery though. So you don't have to leave the browser.

Why aren't there just a simple checkbox that will do that? Well, there is a thought behind that madness. Adding features is easy. Adding more tabs with more checkboxes and sliders, not difficult. But then we would only compete with all other webui's over who can add the most things.

What we are trying to do is adding things without making the user interface bloated. We experiment a lot with the goal to make things as un-bloated as possible. Which makes us do things differently. :) ...and our webui can upscale any image you have, not just newly generated ones. :)

**Comment by Eleksar387** [comment]
Thx! Didn't know that I can just drag the image in here.
I understand your idea behind it. Maybe in the future you could add settings for advanced users. But I am happy for what you are doing here! Keep the good work going.

**Comment by StanleyPain** [comment]
Would love to see a "one button upscale". When creating a large batch of images it really sucks to have to have to upscale them all manually :(. Would be a huge QoL benefit :)
